### PR TITLE
gallery tweaks

### PR DESCRIPTION
**Pull request by maryhipp**
* if auto-switch setting is on, navigate back to images tab on generation (weird behavior if they are on assets tab)
* load images for assets tab is assets tab is active when UI loads 
* handle case where user has zero images and skeleton loader didn't disappear
* TODO: get virtuoso grid to fill viewport if 20 images that are initially loaded does not fill - I couldn't get that working

### CODE REVIEW

File: invokeai/frontend/web/src/features/gallery/components/ImageGalleryContent.tsx [hunk 1]
```diff
@@ -185,11 +185,12 @@ const ImageGalleryContent = () => {
   useEffect(() => {
     dispatch(
       receivedPageOfImages({
-        categories: ['general'],
+        categories,
         is_intermediate: false,
       })
     );
-  }, [dispatch]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (maryhipp) [inline invokeai/frontend/web/src/features/gallery/components/ImageGalleryContent.tsx:192]: @psychedelicious is there a better way to run something only on mount?

Comment (psychedelicious) [reply invokeai/frontend/web/src/features/gallery/components/ImageGalleryContent.tsx:192]: Nope that's how you do it. Could also have a conditional to check if there are already images loaded and not make the request if so.

**Comment by psychedelicious** [comment]
[feat(ui): fill up gallery on app start](https://github.com/invoke-ai/InvokeAI/pull/3682/commits/f79a3bd781501276226aecc20532b6c13d0541a5)
[f79a3bd](https://github.com/invoke-ai/InvokeAI/pull/3682/commits/f79a3bd781501276226aecc20532b6c13d0541a5)

taking the coward's way out on this and just fetching 100 images & 100 assets on app start...

- add `appStarted` action, dispatched once on mount in App.tsx. listener fetches 100 images & 100 assets
- fix bug with selectedBoardId & assets tab

**Review by psychedelicious — APPROVED** [review]

**Comment by psychedelicious** [comment]
@blessedcoolant mind giving this one a go? try with no db to emulate new user experience

**Comment by blessedcoolant** [comment]
@psychedelicious Tested all mentioned conditions. Seems good. Couldn't find any obvious bugs.

**Review by blessedcoolant — APPROVED** [review]

**Pull request merged**